src/features/Classes/InstructorCard/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { format } from 'date-fns';
+import { useSelector } from 'react-redux';
+import { useParams, useLocation } from 'react-router-dom';
+
+import instructorDefaultImage from 'assets/avatar.svg';
+
+import 'features/Classes/InstructorCard/index.scss';
+
+const InstructorCard = () => {
+  const location = useLocation();
+  const { courseId, classId } = useParams();
+  const classes = useSelector((state) => state.classes.table.data);
+
+  const queryParams = new URLSearchParams(location.search);
+  const classIdQuery = queryParams.get('classId');
+
+  const [classInfo] = classes.filter(
+    (classElement) => classElement.classId === classIdQuery?.replaceAll(' ', '+'),
+  );
+
+  const totalEnrolled = (classInfo?.numberOfStudents || 0)
+    + (classInfo?.numberOfPendingStudents || 0);
+
+  const extraInstructors = classInfo?.instructors?.length === 1
+    ? (classInfo?.instructors?.length || 0)
+    : (classInfo?.instructors?.length || 0) - 1;
+
+  const formatDateRange = (startDate, endDate) => {
```

Comment:
Please create a utils file and move this function there so can be used in others components